### ISSUE TITLE
<DropdownMenuTrigger/> missing role attribute

### ISSUE THREAD

**Issue by halid96** [radix-ui/primitives#3490]
<DropdownMenuTrigger /> renders a <button>. This element, I think, should have a role "button" attribute by default. I added this, and the page accessibility score increased.
